### PR TITLE
add routes-google branch for google

### PR DESCRIPTION
**Pull request by HammamAbuShehadeh**
#11 add google route

### CODE REVIEW

**Comment by HammamAbuShehadeh** [comment]
Done Update scheme models files

File: server/server/routes/googleRoute.js
Comment (HammamAbuShehadeh) [inline server/server/routes/googleRoute.js:1]: Looks good thx

File: server/server/routes/googleRoute.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+/********************************ramah */
```
Comment (HammamAbuShehadeh) [inline server/server/routes/googleRoute.js:1]: you should remove unnecessary comment

File: server/server/routes/googleRoute.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+const express = require("express");
+const router = express.Router();
+const passport = require('passport');
+
+
+// @desc authenticate with Google 
+// @route /auth/google
+router.get('/google', passport.authenticate('google', { scope: ['profile', 'email'] }));
```
Comment (HammamAbuShehadeh) [inline server/server/routes/googleRoute.js:9]: looks good thx

File: server/server/routes/googleRoute.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+// @desc Google auth callback
+// @route /auth/google/callback
+router.get('/google/callback', passport.authenticate('google', { failureRedirect: '/user' }), (_, res) => {
+  res.redirect('/test');
+}); 
+
+module.exports = router;
```
Comment (HammamAbuShehadeh) [inline server/server/routes/googleRoute.js:17]: nice work

**Review by HammamAbuShehadeh — COMMENTED** [review]
Looks good thx

File: server/server/routes/googleRoute.js
Comment (ramah-madi) [reply server/server/routes/googleRoute.js:1]: You're welcome!

File: server/server/routes/googleRoute.js (hunk 1 above)
Comment (ramah-madi) [reply server/server/routes/googleRoute.js:1]: Oh ok, I will thx :)

File: server/server/routes/googleRoute.js (hunk 2 above)
Comment (ramah-madi) [reply server/server/routes/googleRoute.js:9]: You're most welcome!

File: server/server/routes/googleRoute.js (hunk 3 above)
Comment (ramah-madi) [reply server/server/routes/googleRoute.js:17]: Thank you ^_^

**Pull request merged**